UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/Shared/_Layout.cshtml

```diff
@@ -5,39 +5,60 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>@ViewData["Title"] - UKHO.MaritimeSafetyInformation.Web</title>
     <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.min.css" />
-    <link rel="stylesheet" href="~/css/site.css" asp-append-version="true" />
+    <link rel="stylesheet" type="text/css" href="https://unpkg.com/%40ukho/styles@1.3.21/dist/ukho.min.css" asp-append-version="true" />
     <link rel="stylesheet" href="~/UKHO.MaritimeSafetyInformation.Web.styles.css" asp-append-version="true" />
+    <link rel="stylesheet" href="~/css/nm_style.min.css" asp-append-version="true" />
```

Comment:
Please can we consider to have just a single application css file?

Reply:
rename scss file as per suggested